ansible/roles/jitsi/tasks/main.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Download and add Prosody Debian packages key
+  ansible.builtin.get_url:
+    url: https://prosody.im/files/prosody-debian-packages.key
+    dest: /etc/apt/keyrings/prosody-debian-packages.key
+    mode: '0644'
+  become: true
+  tags:
+    - role::jitsi
+
+- name: Add Prosody repository to sources list
+  ansible.builtin.apt_repository:
+    repo: "deb [signed-by=/etc/apt/keyrings/prosody-debian-packages.key] http://packages.prosody.im/debian {{ ansible_distribution_release }} main"
+    filename: prosody-debian-packages
+  become: true
+  tags:
+    - role::jitsi
+
+- name: Install lua5.2
+  ansible.builtin.apt:
+    name: lua5.2
+    state: present
+  become: true
+  tags:
+    - role::jitsi
+
+
+- name: Fetch Jitsi GPG key
+  ansible.builtin.get_url:
+    url: https://download.jitsi.org/jitsi-key.gpg.key
+    dest: /tmp/jitsi-key.gpg.key
+    mode: "u=rw,g=r,o=r"
+  become: true
+  tags:
+    - role::jitsi
+
+- name: Convert GPG key to keyring format
+  ansible.builtin.command:
+    cmd: gpg --dearmor -o /usr/share/keyrings/jitsi-keyring.gpg /tmp/jitsi-key.gpg.key
+    creates: /usr/share/keyrings/jitsi-keyring.gpg
+  become: true
+  tags:
+    - role::jitsi
+
+- name: Clean up temporary GPG key file
+  ansible.builtin.file:
+    path: /tmp/jitsi-key.gpg.key
+    state: absent
+  become: true
+  tags:
+    - role::jitsi
+
+- name: Add Jitsi repository to sources list
+  ansible.builtin.apt_repository:
+    repo: "deb [signed-by=/usr/share/keyrings/jitsi-keyring.gpg] https://download.jitsi.org stable/"
+    filename: jitsi-stable
+  become: true
+  tags:
+    - role::jitsi
+
+- name: Preconfigure debconf settings for Jitsi
+  debconf:
+    name: "{{ item.name }}"
+    question: "{{ item.question }}"
+    value: "{{ item.value }}"
+    vtype: "{{ item.vtype }}"
+  loop: "{{ jitsi_debconf_questions }}"
+  become: true
+  tags:
+    - role::jitsi
+
+
+- name: Install Jitsi
+  ansible.builtin.apt:
+    name: jitsi-meet
+    state: present
+  become: true
+  tags:
+    - role::jitsi
+
+- name: Activate the jitsi server block
+  ansible.builtin.file:
+    src: /etc/nginx/sites-available/jitsi.pydis.wtf.conf
+    path: /etc/nginx/sites-enabled/jitsi.pydis.wtf.conf
+    state: link
```

Comment:
You should use the `reload the nginx service` handler here, it also needs a tag.

Reply:
8c6538a